cora/signal/lss.py

```diff
@@ -715,23 +796,45 @@ class BiasedLSSToMap(task.SingleTask):
     lognormal : bool
         Use a lognormal transform to guarantee that the density is always positive (i.e.
         delta > -1).
+    omega_HI_model : {'crichton2015', 'SKA', 'uniform'}
+        The model to use for Omega_HI. See `Notes` for details.
+    omega_HI : float
+        The value for Omega_HI to use when `omega_HI_model` is set to uniform.
+
+    Notes
+    -----
+    We use two models for the evolution of Omega_HI. A fit from Crichton eVjjt al. 2015
```

Comment:
```suggestion
    We use two models for the evolution of Omega_HI. A fit from Crighton et al. 2015
```

Reply:
Looks like I was trying to select this line and the following two in vim!